core/templates/domain/improvements/ineffective-feedback-loop-task.model.spec.ts

```diff
@@ -119,9 +119,9 @@ describe('Ineffective feedback loop task model', function() {
         entity_id: 'eid',
         entity_version: 1,
         // This throws "Type '"???"' is not assignable to type
-        // '"ineffective_feedback_loop"'." This is because 'task_type'
-        // should be equal to 'ineffective_feedback_loop' but we set it
-        // to an invalid value in order to test validations.
+        // '"ineffective_feedback_loop"'.". We need to suppress this error
```

Comment:
Ditto, here and below.

Reply:
I changed this because it does not follow the pattern required for '@ts-ignore' or '@ts-expect-error' which I am implementing in comment-style check
https://github.com/oppia/oppia/pull/13022/files#diff-ebdbd754a4a10a16d3a71e77edcaa4e50310e3e7e14fbb47a456f5d0d0b4d272R92-R105